pkg/cloud/aws/services/userdata/controlplane.go

```diff
@@ -222,6 +192,10 @@ type ControlPlaneInput struct {
 	ServiceDomain     string
 	ServiceSubnet     string
 	KubernetesVersion string
+
+	// TODO extract these since they contain values from above (not certs)
+	ClusterConfiguration string
```

Comment:
I need to restructure these types/function signatures, I'm not super happy with this